cookbook/integration_litellm.ipynb

```diff
@@ -0,0 +1,231 @@
+{
```

Comment:
<p>add description that explains that the openai integration automatically logs all these calls as generations to langfuse. hyperlink to the docs of the openai integration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/538/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>